avro-serializer/src/main/java/io/confluent/kafka/serializers/subject/v2/SubjectNameStrategy.java

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright 2018 Confluent Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+
+package io.confluent.kafka.serializers.subject.v2;
```

Comment:
Have you thought about naming this something other than `SubjectNameStrategy` or calling this package `strategy`. Calling a package `v2` seems very ... unusual.

Reply:
I had the same comment. Encoding versioning in the package seems pretty unconventional.

Reply:
Not sure what `subject` means in the package. By looking at the package contents maybe a `strategy` package would make sense?

Reply:
we would not be able to change the existing package name where the implementation resides since they are all public configs. We can possibly create a new `strategy` package and include the new interface there. But I would prefer some kind of versioning scheme compared to just a different package name.